src/provider/provider_tracking.c

```diff
@@ -405,7 +405,6 @@ static void check_if_tracker_is_empty(umf_memory_tracker_handle_t hTracker,
                         "left)",
                         n_items);
             }
-            assert(n_items == 0);
```

Comment:
what do you think about logging a message here?

Reply:
We already do that (see just above this assert).